gitlens/gitlens-release-notes-current.md

```diff
@@ -19,6 +19,173 @@ Check out our [Changelog](https://github.com/gitkraken/vscode-gitlens/blob/main/
+
+### Opening Repositories
+
+Opening a repository from a Workspace is as easy as tapping the 'Open Repository in New Window' icon.
+
+_Pro Tip: If you prefer to open the Workspace in your current window, hold down the ALT/OPTION key on a Mac and click 'Open Repo'._
+
+### Converting to Cloud Workspaces
+
+For users who prefer to work in the cloud, GitLens offers the ability to convert your local VSCode workspace into a GitKraken Workspace. Just press the 'Convert to Cloud Workspace' icon, provide a name and a description, and your workspace will be stored in the cloud for your GitKraken user account. This means that your Cloud Workspace will now appear in any of your GitKraken shared services, such as GitKraken Client, GitLens, and the GitKraken CLI.
+
+### Understanding Workspace indicators and colors
+Workspaces also include visual indicators to help you understand their status. For example, a green Workspace with an 'O' symbol indicates that it is open in your current window.
+
+## ✨ Commit Graph Enhancements
+We’ve made notable enhancements to the Commit Graph, improving its usability and visual presentation.
+
+### Layouts
+As mentioned above, we’ve restructured our views with our New View Layouts for greater focus and productivity, including the new GitLens Inspect side bar, the re-focused GitLens side bar, and the slimmed-down Source Control side bar. Each of these new layouts is designed to provide more contextual information and insights into what you're actively working on, enhancing your overall experience and productivity.
+
+### Changes Column
+The Changes column is now visible by default. Previously, this column was hidden by default. No more navigating through settings to enable the Changes column – it's right there when you need it, providing immediate insights into the modifications made in each commit. This change is part of our commitment to making GitLens as user-friendly and accessible as possible.
+
+The changes column now also includes a file count. The file account allows you to see at a glance the number of files involved in each commit, helping you quickly grasp the scale of each change. This new count includes files that have been added, modified, or deleted in each commit.
+
+### Branch/Tag Column
+In our ongoing efforts to enhance the user experience, we've made significant improvements to the tooltips on the Branch/Tag icons. Now, they're more uniform and descriptive, providing clearer and more consistent information. This update will make navigating through your branches and tags easier and more intuitive than ever before.
+
+We've resolved an issue that was sometimes preventing Branch/Tag nodes from expanding when hovered over. This bug fix ensures a smoother and more reliable interaction with the Branch/Tag nodes, enhancing the overall usability and functionality of GitLens. We're continuously working to improve GitLens and your feedback is vital in helping us fix these bugs. Thank you for your support!
```

Comment:
```suggestion
```

This bugfix doesn't warrant a mention in this section imo - in the changelog is enough.